src/components/AddBook.js

```diff
@@ -30,7 +30,7 @@ function AddBook() {
       <input type="text" name="title" placeholder="Book Title" required />
       <input type="text" name="author" placeholder="Author" required />
       <input type="text" name="category" placeholder="Category" required />
-      <button type="submit">ADD BOOK</button>
+      <button className="addbutton" type="submit">ADD BOOK</button>
```

Comment:
- [ OPTIONAL ] Great job working on the functionality but when I add a new book to the list I can't see it till I refresh the page so please try to figure the issue and to solve it to make your app just work in a perfect way 🚀😊🔥💪🏻💯💯

Reply:
If I knew how to do that I would have already done it.

Reply:
-  I think it something in your addbook file when you make your useDispatch maybe you can useEffect to try but also if you search about it you can see alot of solutions I asked for it as optional no worries .😊

Reply:
Yeah, thanks for the suggestions. I've actually spent quite some time trying to figure this out, I'll probably work on it after I am done with all the tasks for the week.